dist/src/main/java/io/camunda/zeebe/shared/security/SecurityConfiguration.java

```diff
@@ -61,6 +62,11 @@ public SecurityFilterChain managementSecurity(final HttpSecurity http) throws Ex
 
   @Bean
   @ConditionalOnRestGatewayEnabled
+  @ConditionalOnProperty(
+      prefix = "zeebe.gateway.security",
+      value = "authentication.mode",
+      havingValue = "identity",
+      matchIfMissing = true)
```

Comment:
```suggestion
  @ConditionalOnProperty(
      name = {"zeebe.gateway.security.authentication.mode", "zeebe.broker.gateway.security.authentication.mode"}
      havingValue = "identity",
      matchIfMissing = true)
```

:question: When the gateway runs embedded, then the authentication mode is set via `zeebe.broker.gateway.security.authentication.mode`. Am I correct?